File: BackendWebApp/WebShopAPI/WebShopAPI.Models/Order.cs
namespace WebShopAPI.Models
{
    public class Order
    {
        public int OrderId { get; set; }
        public string Comment { get; set; }
        public string Address { get; set; }
        public DateTime OrderDate { get; set; }
        //public User UserBuyer { get; set; }
        public int UserBuyerId { get; set; }
        public  string Status { get; set; }
        public DateTime DeliveryTime { get; set; }
        public double TotalAmount { get; set; }
        public List<ProductOrder> OrderProducts { get; set; }
    }
}

File: FrontendWebApp/web-shop-app/src/models/Order.js
import Product from "./Product";

export class Order {
  constructor(
    OrderId = 0,
    Comment = "",
    Address = "",
    OrderDate = null,
    UserBuyerId = 0,
    TotalAmount = 0,
    SellerCount = 0,
    Status = "",
    DeliveryTime = "",
    Products = [],
    
  ) {
    this.orderId = OrderId;
    this.comment = Comment;
    this.address = Address;
    this.orderDate = OrderDate;
    this.userBuyerId = UserBuyerId;
    this.totalAmount = TotalAmount;
    this.sellerCount = SellerCount;
    this.status = Status;
    this.deliveryTime = DeliveryTime;
    this.products = Products;
  }
}
export default Order;
